client/my-sites/stats/wordads/index.jsx

```diff
@@ -245,7 +256,7 @@ class WordAds extends Component {
 								</>
 
 								<div className="stats__module-list stats__module-headerless--unified">
-									<WordAdsEarnings site={ site } />
+									<WordAdsEarnings site={ site } showTotalsSection={ ! showNewHighlights } />
```

Comment:
If you add a condition around this component, you won't have to pass the property. For example:
```
{ !isFeatureFlagEnabled && <WordAdsEarnings site={ site } /> }
```
It might also be better to add the condition around the whole `div` if it doesn't break the layout.

Reply:
We can't conditionally render the WordAdsEarnings component because it also handles the render of up to three tables in addition to the totals section. If we used a feature flag we could do that check inside the component but, as noted above, I can remove this completely now that the new icons are part of the PR (incoming).

Reply:
The old code path has been removed so the property is no longer needed.